### DISCUSSION TITLE
Issue with deployment using Github pages

### DISCUSSION THREAD

**Discussion by vPackets** [chipzoller/hugo-clarity#433] [Q&A] [answered]
Hi everyone !

New to this theme that I like a lot, I was previously using another that was working fine but the migration to clarity is making me struggling..

Usually I was pushing the website webpages to the docs/ folder using the publishDir = "docs" config in the config.toml file. 
This way I could tell GitHub pages to use the main branch and the docs folder but this is not working.

Does anyone knows what could be the issue ? my code is here : https://github.com/vPackets/vPackets-blog

and the blog is supposed to work here : https://github.com/vPackets/vPackets-blog ... Looks like not everything is working :D 

Thanks for the help !

PS: The website works properly locally :/

**Comment by Tekki** [comment, marked as answer]
The baseUrl in your `config.toml` is set to http instead of https. If you check your site in the browser's developer console, you'll see a lot of errors because of that.

**Reply by vPackets** [reply]
Indeed, Looks like there was a problem with that ... That's weird because it was working with another theme.... I ll have a check ! 

Thanks for the help provided ! Much appreciated !
